.github/ISSUE_TEMPLATE/fix-suggestions.md

```diff
@@ -0,0 +1,34 @@
+---
+name: Fix Suggestions
+about: Add a new component with fix suggestions to existing check
+title: "[Fix Suggestions]"
+labels: autofix
+assignees: ''
+
+---
+
+**Which Atlas check is the fix suggestion related to?** 
+
+Name of Atlas check that need to be added the suggested fix component. Link to associate check script
```

Comment:
nit: It would be nice to reword this to "Name and link of the existing Atlas Check that needs to be updated to include the below fix suggestion(s)"